scripts/package.sh

```diff
@@ -140,7 +140,11 @@ package_download_ball()
 {
   pkg_url=$1
   pkg_hash=$2
-  pkg=$SYS_PREFIXROOT/packages/`basename $pkg_url`
+  pkg_basename=$3
+  if [ "$pkg_basename" = "" ]; then
```

Comment:
While this works, I never know if it is better to test with `-z` or use the `"X$2" = "X"` approach we use a lot